File: app/src/main/java/com/example/wechatmoments/HookEntry.java
package com.example.wechatmoments;

import de.robv.android.xposed.IXposedHookLoadPackage;
import de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam;

public class HookEntry implements IXposedHookLoadPackage {
    @Override
    public void handleLoadPackage(LoadPackageParam lpparam) throws Throwable {
        if (!lpparam.packageName.equals("com.tencent.mm")) return;
        // 微信 Hook 入口已建立
    }
}

File: app/src/main/java/com/example/wechatmoments/HookEntry.kt
package com.example.wechatmoments

import de.robv.android.xposed.IXposedHookLoadPackage
import de.robv.android.xposed.callbacks.XC_LoadPackage.LoadPackageParam

class HookEntry : IXposedHookLoadPackage {
    override fun handleLoadPackage(lpparam: LoadPackageParam) {
        if (lpparam.packageName != "com.tencent.mm") return
        // 微信 Hook 入口代码已初始化
    }
}
